server/paymentProviders/stripe/webhook.ts

```diff
@@ -614,22 +632,21 @@ export const webhook = async (request: Request<unknown, Stripe.Event>) => {
     return Promise.resolve();
   }
 
-  event = await validateWebhook(request);
+  if (event.type.startsWith('issuing')) {
+    return handleIssuingWebhooks(request);
```

Comment:
We may receive unexpected events in `handleIssuingWebhooks` that we may fail to parse due to an undefined virtualCardId.